HousingManagementSystemApi.Tests/GatewaysTests/PropertiesGatewayTests.cs

```diff
@@ -0,0 +1,156 @@
+namespace HousingManagementSystemApi.Tests.GatewaysTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Gateways;
+    using Moq;
+    using RichardSzalay.MockHttp;
+    using Xunit;
+
+    public class PropertiesGatewayTests
+    {
+        private readonly PropertiesGateway systemUnderTest;
+
+        private const string Postcode = "E100 1QZ";
+        private string propertiesSearchResponse =
+            @"[
+  {
+    ""propRef"": ""100000000001"",
+    ""postCode"": ""E100 1QZ"",
+    ""address1"": ""1A Old Hackney Road""
+  }
+]";
+
+        private const string PostcodeMultipleAddresses = "E100 1QQ";
+        private string propertiesSearchResponse2 =
```

Comment:
Can we rename this to propertiesSearchResponseMultipleAddresses. I think it would make the tests clearer when reading though them